fineract-provider/src/main/java/org/apache/fineract/infrastructure/event/external/service/serialization/mapper/loan/LoanAccountDataMapper.java

```diff
@@ -36,5 +36,6 @@ public interface LoanAccountDataMapper {
     @Mapping(source = "interestRecalculationEnabled", target = "isInterestRecalculationEnabled")
     LoanAccountDataV1 map(LoanAccountData source);
 
+    @Mapping(target = "unpaidCharges", ignore = true)
```

Comment:
I don't think we even need this method here.
We could just modify the annotation on the class to this:
```
@Mapper(config = AvroMapperConfig.class, uses = { LoanTransactionDataMapper.class })
```

Reply:
I give a try :)